docs/streams/stream-examples/realtime-updates.md

```diff
@@ -1,33 +1,62 @@
 ---
 sidebar_position: 140
-title: Realtime DB Updates Example
+title: Real-Time DB Updates Example
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import Prerequisites from '../../_partials/_prerequisites-sdk-api-key.md';
+import Steps from '../../_partials/_get-started-steps.md';
+import ConnectToGDN from '../../_partials/_connect-to-gdn-code-block.md';
 
-This tutorial is about using Macrometa GDN as a realtime database with local latencies across the globe.
+This tutorial demonstrates how to use Macrometa GDN as a real-time database with local latencies across the globe.
 
-## Prerequisites
+<Prerequisites />
 
-- A Macrometa account with sufficient permissions to create streams.
-- An API key. For more information, refer to [Create API Keys](../../account-management/api-keys/create-api-keys).
-- Appropriate SDK installed. For more information, refer to [Install SDKs](../../sdks/install-sdks.md).
+## Step-by-Step Instructions
 
-## Realtime Updates Code
+This page guides you through creating a collection, subscribing to the collection, and automatically adding and deleting data to the collection.
+
+<Steps />
+
+If you want to skip the explanation and just run the code, then go directly to the [Full Demo File](#full-demo-file).
+
+### Step 1. Connect to GDN
+
+To update a database with Macrometa Global Data Network (GDN), you must first establish a connection to a local region.
+
+<ConnectToGDN />
+
+### Step 2. Set Variables
+
+Set the variables required to run the code.
 
 <Tabs groupId="operating-systems">
-<TabItem value="js" label="Javascript">
+<TabItem value="py" label="Python SDK">
 
-```js
-const jsc8 = require("jsc8");
+```py
+import threading
+import pprint
+import time
+COLLECTION_NAME = "ddos"
 
-// Constants - DB
-const globalUrl = "https://play.paas.macrometa.io/";
-const email = "your@email.com";
-const password = "password";
-const client = new jsc8(globalUrl);
+# Variables - Data
```

Comment:
```suggestion
# Variables
```

Reply:
Very minor suggestion - I like to have the comments in the different versions exactly match wherever possible. If you apply this, then also update the full demo.